src/server/authMiddleware.ts

```diff
@@ -72,19 +95,32 @@ const createAuthMiddleware =
 	async (req: NextRequest) => {
 		console.debug('Auth middleware starts');
 
+		const { publicRoutes, privateRoutes, ...restOptions } = options;
+		if (publicRoutes && privateRoutes) {
```

Comment:
this line don't consider a case here that public routes / private routes are empty array, which will yield true (like the file above)

we should handle this here as well